conf/monitoring/carbon.conf.example

```diff
@@ -38,7 +38,7 @@ LOG_DIR        = %%install_dir%%/logs/
 PID_DIR        = %%install_dir%%/var/run/
 
 # Enable daily log rotation. If disabled, a kill -HUP can be used after a manual rotate
-ENABLE_LOGROTATION = False
+ENABLE_LOGROTATION = True
```

Comment:
Bad rebase.
I need to change it in three places.
Will be part of a separate fix.